### PR TITLE
Put some of sample ‌BitTextField in separate sections in demo page (#1643)

### LINKED ISSUE #1643: Separate samples of the BitTextField component in the demo page
Some of the BitTextFiled different samples have been provided in [one section.](https://bitcomponents.azurewebsites.net/components/text-field#example1)
It is hard for a user to find a certain sample and copy its code.
So, the samples should be provided in separate sections.
freesnippingtool com_capture_20220310165831

### PR DESCRIPTION
**Pull request by saeedhalab**
This closes #1643

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 <ComponentDemo ComponentName="TextField" ComponentDescription="Text fields give people a way to enter and edit text. They’re used in forms, modal dialogs, tables, and other surfaces where text input is required."
                ComponentParameters="componentParameters" EnumParameters="enumParameters">
-    <ComponentExampleBox Title="BitTextField" HTMLSourceCode="@example1HTMLCode" CSharpSourceCode="@example1CSharpCode" ExampleId="example1">
+
+    <ComponentExampleBox Title="Standard TextField"
+                         HTMLSourceCode="@example1HTMLCode"
+                         CSharpSourceCode="@example1CSharpCode"
+                         ExampleId="example2">
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor:9]: This example Id must be "example1"

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor [hunk 2]
```diff
@@ -12,27 +16,77 @@
...
+                         HTMLSourceCode="@example2HTMLCode"
+                         ExampleId="example2">
+        <ExamplePreview>
             <div>
                 <BitTextField Label="Read-only" IsReadonly="true" DefaultValue="I am read-only"></BitTextField>
             </div>
+        </ExamplePreview>
+    </ComponentExampleBox>
+
+    <ComponentExampleBox Title="With Placeholder TextField"
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor:32]: The title of this example must be "TextField With Placeholder"

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor [hunk 3]
```diff
@@ -12,27 +16,77 @@
...
+                         HTMLSourceCode="@example5HTMLCode" 
+                         ExampleId="example5">
+        <ExamplePreview>
             <div>
                 <BitTextField Label="Controlled TextField Limiting Length Of Value To 10" MaxLength="10"></BitTextField>
             </div>
+        </ExamplePreview>
+    </ComponentExampleBox>
+
+    <ComponentExampleBox Title="With An Icon TextField"
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor:65]: The title of this example must be "TextField With An Icon"

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor [hunk 4]
```diff
@@ -12,27 +16,77 @@
...
+                         HTMLSourceCode="@example6HTMLCode" 
+                         ExampleId="example6">
+        <ExamplePreview>
             <div>
                 <BitTextField Label="With An Icon" IconName="BitIconName.CalendarMirrored"></BitTextField>
             </div>
+        </ExamplePreview>
+    </ComponentExampleBox>
+
+    <ComponentExampleBox Title="Password With Reveal Button TextField"
```
Comment (maryamhdr) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor:75]: The title of this example must be "Password TextField With Reveal Button"

**Review by maryamhdr — CHANGES_REQUESTED** [review]

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**